lib/iris/tests/integration/test_netcdf.py

```diff
@@ -214,7 +215,9 @@ def test_lazy_mask_preserve_fill_value(self):
             ncube = iris.load_cube(filename)
             # Lazy save of the masked cube
             iris.save(ncube, other_filename, unlimited_dimensions=[])
-            self.assertCDL(other_filename)
+            ds = nc.Dataset(other_filename, 'r')
+            avar = ds['unknown']
+            self.assertEqual(avar._FillValue, -1)
```

Comment:
@marqh Remove the associated expected CDL file for this test i.e. `iris/tests/results/integration/netcdf/TestLazySave/lazy_mask_preserve_fill_value.cdl`

That's also going to apply to all the other tests that you touched, thanks!